playbook.md

```diff
@@ -168,6 +140,34 @@ regularly assessing the impact of all our policies and practices.
 
 Thank you for being you - it makes us better.
 
+### Advisory Council
+
+dxw does not have external investors or shareholders. To help keep us honest, we are very lucky to have a voluntary advisory council, made up of a small group of senior figures, who meet quarterly to discuss and provide guidance on our strategy, business plan and other topics related to company health and progression. The group have a wealth of experience of public and private sector work, as well as being friends of dxw that share our values.
+
+### This Playbook
```

Comment:
```suggestion
## This Playbook
```

I think this belongs at a level up?

Reply:
I remember debating this last year. I'd prefer to leave the bit about the Playbook under the Who we are section rather than have a top-level section all of its own.

Reply:
It's not where I'd look for information about the playbook, but I guess we can hash that out separately.